.gitignore

```diff
@@ -381,3 +381,4 @@ airport-app-backend
 golang-app-starter
 
 .env
+flyway/conf/flyway.conf
```

Comment:
why is this ignored? wouldn't it be useful to share amongst the devs?

Reply:
the file has db credentials just like .env